### ISSUE TITLE
Is the dependency on re-linq going away?

### ISSUE THREAD

**Issue by IanKemp** [dotnet/efcore#17397]
* [The package page](https://www.nuget.org/packages/Remotion.Linq/) refers to a now-defunct CodePlex site, the package has no .NET Standard 2.0 target, and no new version of the package has been published since February last year.
* [The repo](https://github.com/re-motion/Relinq) has had no commits in over a year.
* [The issue board](https://re-motion.atlassian.net/projects/RMLNQ/issues/) has had no activity since January.

tl;dr it looks like re-linq is a dead project.

Having a dependency on a dead project is a significant risk, especially for a widely-used and heavily depended on project like EF Core. Are there any plans to remove this dependency going forward?

**Comment by tuespetre** [comment]
@IanKemp the team has been working to rebuild the entire query compiler from the ground up for 3.0. Can't wait to see their 3.0 release 😄

**Comment by ajcvickers** [comment]
@IanKemp  See https://github.com/aspnet/EntityFrameworkCore/issues/12795#issuecomment-419469371
